packages/module/src/WarningModal/WarningModal.tsx

```diff
@@ -69,17 +90,36 @@ const WarningModal: React.FunctionComponent<WarningModalProps> = ({
       ouiaId={ouiaId}
       {...props}
     >
-      {children}
-      {withCheckbox ? (
-        <Checkbox
-          isChecked={checked}
-          onChange={(_event, value) => setChecked(value)}
-          label={checkboxLabel}
-          id="warning-modal-check"
-          className="pf-v6-u-mt-lg"
-          ouiaId={`${ouiaId}-confirm-checkbox`}
-        />
-      ) : null}
+      <Stack hasGutter>
+        <StackItem>{children}</StackItem>
+        <StackItem>
+          {confirmationInputProps ? (
```

Comment:
```suggestion
          {confirmationText ? (
```
we probably want to show the confirmation input only if the text is passed, not necessarily the input props

Reply:
fixed